wmake

```diff
@@ -50,6 +50,9 @@ _linux_setup() {
         exit 1;
     fi
 
+    printf "Initiating submodules...\n"
+    git submodule update --init
```

Comment:
Would this not be better `git submodule update --init --recursive`?

Reply:
true tm